src/main/java/com/ashkan/moviesapi/entities/Movie.java

```diff
@@ -1,13 +1,20 @@
-package com.ashkan.moviesapi.entity;
+package com.ashkan.moviesapi.entities;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
 import java.util.Collection;
 import java.util.Objects;
 
 @Entity
+@JsonIgnoreProperties(value={ "id","description","registeringUser",
```

Comment:
I think you are doing this because you are using this Entity classes as objects to be used in the Controllers, that is fine.
Usually the objects we receive as payloads don't have the same structure as the Entities because the Clients needs maybe simpler objects, that is why you can use DTOs objects to work in the Controllers and use the Entities as Persitence objects because that is its main goal.
Please analyze if this could be a good suggestion for your project.

Reply:
I think its a good suggestion for the project ,also will solve the next problem